src/pages/en/guides/server-side-rendering.md

```diff
@@ -132,3 +139,75 @@ export async function get({ params }) {
   });
 }
 ```
+
+In this example, a valid HTML code can be returned to render the whole page or some of it's content.
+
+### Serverless Functions
+
+API routes can be used to run [serverless functions](https://en.wikipedia.org/wiki/Serverless_computing#Serverless_runtimes) (e.g. authentications, database access, and verifications) without exposing sensitive data to the client.
+
+In the example below, an API route is used to verify Google reCaptcha v3 without exposing the site-secret to the clients.
```

Comment:
```suggestion
In addition to content fetching and server-side rendering, API routes can be used as REST API endpoints to run [serverless functions](https://en.wikipedia.org/wiki/Serverless_computing#Serverless_runtimes) (e.g. authentications, database access, and verifications) without exposing sensitive data to the client.

In the example below, an API route is used to verify Google reCaptcha v3 without exposing the site-secret to the clients.
```
Added per @lostra01 's request, since he's not at the computer and easily able to do it right now.  😄